coalib/settings/ConfigurationGathering.py

```diff
@@ -283,6 +287,14 @@ def load_configuration(arg_list,
 
         sections = merge_section_dicts(sections, cli_sections)
 
+    if exclude_sections is not None:
```

Comment:
I think this `if` is useless, since the loop will not run anyway if `exclude_sections` are empty.

Reply:
Are the changes fine now?

Reply:
Yeah, just wait for other reviewers that has write access :)

Reply:
Thanks a lot :)